app/client/pages/course/courseSummary.js

```diff
@@ -0,0 +1,220 @@
+import Typography from '@mui/material/Typography';
+import Grid from '@mui/material/Grid';
+import TableHead from '@mui/material/TableHead';
+import TableRow from '@mui/material/TableRow';
+import Paper from '@mui/material/Paper';
+import Container from '@mui/material/Container';
+import Button from '@mui/material/Button';
+
+
+
+const post = {
+  'id': '634cdb493619c990775bac2f',
+  'title': 'Data Science Fundamentals with Python',
+  'course_info': 'Data Science is one ofData Science is one of Data Science is one of the hottest professions of the decade, and the demand for d Data Science is one of the hottest professions of the decade, and the demand for data scientists Data the hottest professions of the decade, and the demand for data scientists Data Science is one of the hottest professions of the decade, and the demand for data scientists',
+  'rating': 4.2,
+  'lecturer': {
+    'id': '634cdce8f7505e71322a9663',
+    'name': 'Andkew MG',
+    'email': 'andkew.mg@boun.edu.tr', // for send email button
+    'is_confirmed': true,
+    'image': 'https://buacademy.com/static/image/634cdce8f7505e71322a9663'
+  },
+  'tags': ['data science', 'computer science', 'python'],
+  'chapters': [
+    {
+      'id': '634cde32693cd0c82806e64c',
+      'chapter_name': 'Introduction & Descriptive Statistics',
+    },
+    {
+      'id': '634cde8a9dbd91c3a2c97324',
+      'chapter_name': 'Data Visualization',
+    },
+    {
+      'id': '634cde8fc2f25e1a7694d2f6',
+      'chapter_name': 'Introduction to Probability Distributions',
+    },
+    {
+      'id': '634cde32693cd0c82806e64c',
+      'chapter_name': 'Introduction & Descriptive Statistics',
+    },
+    {
+      'id': '634cde8a9dbd91c3a2c97324',
+      'chapter_name': 'Data Visualization',
+    },
+    {
+      'id': '634cde8fc2f25e1a7694d2f6',
+      'chapter_name': 'Introduction to Probability Distributions',
+    },
+    {
+      'id': '634cde32693cd0c82806e64c',
+      'chapter_name': 'Introduction & Descriptive Statistics',
+    },
+    {
+      'id': '634cde8a9dbd91c3a2c97324',
+      'chapter_name': 'Data Visualization',
+    },
+    {
+      'id': '634cde8fc2f25e1a7694d2f6',
+      'chapter_name': 'Introduction to Probability Distributions',
+    },
+  ],
+  'image': 'https://cdn.educba.com/academy/wp-content/uploads/2019/03/Introduction-To-Data-Science.jpg',
+  'enrolled': true
+}
+
+
+
+export default function courseSummary() {
+
+
+  post.chapters.map((chapter) => {
+    console.log(chapter.chapter_name)
+  })
+
+  return (
+    < >
+      <Grid sx={{ width: '100%', minHeight: 250, 'background-color': '#3D8BE0', height: 250, 'padding-top': 20 }}  >
+        <Grid  sx={{ height: '95%', width: '80%', margin: 'auto' }}>
+          < Grid container sx={{ float: 'left', width: '60%', height: '100%', padding: 3, 'grid-row-gap': 20}}>
+            <Grid item xs={12} sx={{ margin: 'auto' }}>
+              <Typography variant="h3"   >
+                {post.title}
+              </Typography>
```

Comment:
General comment for text: Try to create contrast between the text and background color. For instance, the description and chapters are very nice since black has high contrast with white. However, page header text is black and it does not have high contrast with blue background.